pkg/controller/controller_binding.go

```diff
@@ -600,14 +644,25 @@ func (c *controller) reconcileServiceInstanceCredential(binding *v1alpha1.Servic
 				"Unbind call failed. "+s)
 
 			if !time.Now().Before(toUpdate.Status.OperationStartTime.Time.Add(c.reconciliationRetryDuration)) {
-				s := fmt.Sprintf(`Stopping reconciliation retries on ServiceInstanceCredential "%v/%v" because too much time has elapsed`, binding.Namespace, binding.Name)
-				glog.Info(s)
-				c.recorder.Event(binding, api.EventTypeWarning, errorReconciliationRetryTimeoutReason, s)
-				c.setServiceInstanceCredentialCondition(toUpdate,
-					v1alpha1.ServiceInstanceCredentialConditionFailed,
-					v1alpha1.ConditionTrue,
-					errorReconciliationRetryTimeoutReason,
-					s)
+				if toUpdate.Status.OrphanMitigationInProgress {
+					s := fmt.Sprintf(`Stopping reconciliation retries on ServiceInstanceCredential "%v/%v" because too much time has elapsed during orphan mitigation`, binding.Namespace, binding.Name)
+					glog.Info(s)
+					c.recorder.Event(binding, api.EventTypeWarning, errorReconciliationRetryTimeoutReason, s)
+					c.setServiceInstanceCredentialCondition(toUpdate,
```

Comment:
Don't want to overwrite the Failed condition when orphan mitigation fails. Preserve the original Failed condition, which we should set with the reason why orphan mitigation was needed when orphan mitigation starts.